### PR TITLE
 feat: implement feed view

### PR DESCRIPTION
**Pull request by aman-singh7**

### CODE REVIEW

File: lib/presentation/feed/widgets/feed_card.dart [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+                          ),
+                          style: Theme.of(context)
+                              .textTheme
+                              .headline6
+                              ?.copyWith(fontSize: 12.sp),
+                          overflow: TextOverflow.ellipsis,
+                        ),
+                      ),
+                      const Spacer(),
+                      ValueListenableBuilder<bool>(
```
Comment (praeclarumjj3) [inline lib/presentation/feed/widgets/feed_card.dart:131]: What's its purpose here?

File: lib/presentation/feed/feed_screen.dart [hunk 2]
```diff
@@ -1,12 +1,84 @@
...
+                return SizedBox(
+                  height: 80,
+                  child: Center(
+                    child: Visibility(
+                      visible: state.allLoaded,
+                      replacement: const CircularProgressIndicator(
+                        color: AppColors.primary,
+                      ),
+                      child: const Text(
+                        'Nothing more to show!!',
```
Comment (praeclarumjj3) [inline lib/presentation/feed/feed_screen.dart:68]: This is never displayed.

**Review by praeclarumjj3 — CHANGES_REQUESTED** [review]

File: lib/presentation/feed/widgets/feed_card.dart (hunk 1 above)
Comment (aman-singh7) [reply lib/presentation/feed/widgets/feed_card.dart:131]: This widget will build the icon according to the current state of the ExpansionTile.

**Comment by praeclarumjj3** [comment]
@aman-singh7 Please fix the conflicts, and then I will merge.

**Comment by aman-singh7** [comment]
@praeclarumjj3 merge conflicts are fixed.

**Pull request merged**